package.json

```diff
@@ -1,97 +1,88 @@
-		"react-icons": "^4.3.1",
-		"react-landing-page": "^2.0.2",
-		"react-reveal": "^1.2.2",
-		"rebass": "^3.0.0-12",
-		"styled-components": "^5.3.3",
-		"passport": "^0.5.2",
-		"passport-local": "^1.0.0",
-		"pg": "^8.7.3"
-	},
-	"engines": {
-		"node": "^12 || ^14 || ^16",
-		"npm": "^6 || ^7 || ^8"
-	}
+  "name": "starter-kit",
+  "version": "0.0.1",
+  "description": "Starter kit for full-stack JavaScript projects",
+  "private": true,
+  "scripts": {
+    "prebuild": "npm run cleanup",
+    "build": "npm run build:server && npm run build:client",
+    "build:client": "webpack --config client/webpack/prod.config.js",
+    "build:server": "babel server --out-dir dist",
+    "cleanup": "rimraf ./dist/*",
+    "dev": "concurrently -k -n \"client,server\" \"npm run dev:client\" \"npm run dev:server\"",
+    "dev:client": "webpack serve --config client/webpack/dev.config.js",
+    "dev:server": "cross-env PORT=3300 nodemon --exitcrash --inspect --delay 500ms --watch server --exec babel-node server/server.js",
+    "dev:server-win": "npm run build:server && npm run start",
+    "lint": "npm run lint:eslint && npm run lint:prettier -- --check",
+    "preserve": "npm run build",
+    "serve": "npm start",
+    "start": "node dist/server.js",
+    "lint:eslint": "eslint .",
+    "lint:fix": "npm run lint:eslint -- --fix && npm run lint:prettier -- --write",
+    "lint:prettier": "prettier ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/textbook/starter-kit.git"
+  },
+  "author": "Jonathan Sharpe <mail@jonrshar.pe>",
+  "contributors": [],
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/textbook/starter-kit/issues"
+  },
+  "homepage": "https://github.com/textbook/starter-kit#readme",
+  "devDependencies": {
+    "@babel/cli": "^7.17.0",
+    "@babel/core": "^7.17.0",
+    "@babel/eslint-parser": "^7.17.0",
+    "@babel/node": "^7.16.8",
+    "@babel/plugin-proposal-class-properties": "^7.16.7",
+    "@babel/plugin-transform-runtime": "^7.17.0",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-react": "^7.16.7",
+    "@codeyourfuture/eslint-config-standard": "^3.0.3",
+    "babel-loader": "^8.2.3",
+    "concurrently": "^7.0.0",
+    "cross-env": "^7.0.3",
+    "css-loader": "^6.6.0",
+    "eslint": "^8.8.0",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-react": "^7.28.0",
+    "eslint-plugin-react-hooks": "^4.3.0",
+    "file-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.5.0",
+    "html-webpack-tags-plugin": "^3.0.2",
+    "nodemon": "^2.0.15",
+    "prettier": "^2.5.1",
+    "prop-types": "^15.8.1",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-router-dom": "^6.2.1",
+    "rimraf": "^3.0.2",
+    "style-loader": "^3.3.1",
+    "wait-on": "^6.0.0",
+    "webpack": "^5.68.0",
+    "webpack-cli": "^4.9.2",
+    "webpack-dev-server": "^4.7.4",
+    "webpack-merge": "^5.8.0"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.17.0",
```

Comment:
I think something has gone wrong here as we see to have removed a lot of the dependencies. Was this intentional?

Reply:
I am looking into it now. I accepted the new changes, did not intend to remove any dependencies.